streams/javahippie.json

```diff
@@ -0,0 +1,26 @@
+{
+  "twitch_handle": "javahippie",
+  "youtube_handle": "javahippie",
+  "twitter_handle": "javahippie",
+  "streamer_timezone": "CEST",
+  "schedule": [
+    {
+      "title": "BreakPoint - Functional Friday Lunch Break",
+      "start_date_utc": "2020-08-28T10:30",
+      "duration_in_minutes": 30,
+      "platform": "twitch",
```

Comment:
Should be optional in case I set just one platform above.

Reply:
That's right. Also, maybe should be an array? Do we have people using restream.io or sth. similar?

Reply:
Don't think anybody uses it, but we shouldn't exclude that option. To make data entry easier, I would prefer a second property `platforms` that takes a string array. Then the code building the site can use whatever is present.

Reply:
On second thought, having both "platform" and "stream link" is redundant. What would be a better construct here? Maybe an array of object?

```json
{
  "platform": "youtube", //required
  "stream_link": "..." //optional
}
```

Reply:
I like that suggestion. An array of objects like that where if the URL is missing then it's inferred from the top-level `XXX_handle`.

```
"stream_links": [
  {"platform": "twitch"},  // <-- inferred URL: https://twitch.tv/${twitch_handle}
  {"platform": "twitch", "url": "https://twitch.tv/my-colleague"}
]
```

I do occasionally stream on 2 channels at once, BTW. When I'm pairing with a colleague we might both stream it.

Reply:
That's a good solution! I added it to the example and the schema